### ISSUE TITLE
Mute characters should at least have a basic way of communicating.

### ISSUE THREAD

**Issue by actually-the-artificer** [CleverRaven/Cataclysm-DDA#63584]
### Is your feature request related to a problem? Please describe.

I am always frustrated when my Mute character cannot communicate with NPCs on a rudimentary level. The feature was recently PR'd with the Churl, so why not allow Mute characters a similar way as well?

### Solution you would like.

Instead of locking a Mute character out of communication entirely, a Mute character should be able to use a phone/tablet/laptop computer to communicate with text-to-speech. Another way if the character has not yet found these devices is to use pen & paper.

### Describe alternatives you have considered.

Sign language is another option, but in the whole of the USA, less than 1% of people know sign language - 250,000 to 500,000 individuals. That number would be severely diminished to be vanishingly low Post-Cataclysm.

### Additional context

This feature will also allow a character to do something which someone would do in real life. There are multiple apps for smartphones that can do text-to-speech, and it is certain that someone who is mute would use it to speak when sign language is not understood by another party.

**Comment by RenechCDDA** [comment]
You have it backwards. The recent PR to churls (still open; not merged) *adds restrictions* to their ability to communicate.

The entire point of the mute trait is the inability to communicate with NPCs. If you find this frustrating, *do not pick the mute trait*.

**Comment by KHeket** [comment]
> You have it backwards. The recent PR to churls (still open; not merged) _adds restrictions_ to their ability to communicate.
> 
> The entire point of the mute trait is the inability to communicate with NPCs. If you find this frustrating, _do not pick the mute trait_.

This game is about realism, not about "If you take trait - your character magically cant do simple things". May be is will be ok for magiclysm as a magick curse, but not for non magic worlds like vanilla. So, if deaf/mute people can do such simple thing in real world - they should have ability to make it in game
And author of pr describe good solutions, I also want to add, that it will be good, if the distance of communication with phone/tablet/laptop computer/pen&paper will be reduced to 1-2 tiles, like in real world its will be very hard to read little letters on big distance

**Comment by anoobindisguise** [comment]
> This game is about realism, not about "If you take trait - your character magically cant do simple things"

This isn't necessarily true. There are certain traits that enforce specific behaviors, such as hates vegetables forcing a dietary restriction for no reason other than "veggie icky" and not a dietary restriction. Illiterate characters could have fun reading a comic book and could probably understand the Exodii blueprints, but the trait makes for better gameplay if it hard-disables books rather than having many special cases. Wayfarer prevents vehicle use. Brawler prevents guns and bows. Etc. It's probably OK for Mute to in game be a "your access to NPCs is totally disabled" trait.

**Comment by MochaModifier** [comment]
> > This game is about realism, not about "If you take trait - your character magically cant do simple things"
> 
> This isn't necessarily true. There are certain traits that enforce specific behaviors, such as hates vegetables forcing a dietary restriction for no reason other than "veggie icky" and not an allergy. Illiterate characters could have fun reading a comic book and could probably understand the Exodii blueprints, but the trait makes for better gameplay if it hard-disables books rather than having many special cases. Wayfarer prevents vehicle use. Brawler prevents guns and bows. Etc. It's probably OK for Mute to in game be a "your access to NPCs is totally disabled" trait.

I think the issue however, is that unlike the personal or traumatic reason that has forced a player character to not consume vegetables, use a gun, or ride a vehicle - being mute or deaf means other NPCs are refusing to deal with the player character.

It would be totally acceptable if this was a trait about being anti-social and refusing to engage with other people - but this is a trait that is based on a real disability, a disability which people have learned to live with and interact meaningfully with their community daily.

Making mute and deaf traits that better identify to, and represent how a survivor would manage in the cataclysm in social interactions can still fill the role of a challenging negative trait - but still manageable in the way you can find corrective eyewear, use an umbrella and cover up, etc.

A new trait that is simply "anti-social" for characters that refuse to interact with NPCs is easily justifiable in the way a brawler refuses to use guns/bows. This could describe either a behavioral trait that isn't strictly associated to a particular disability (which granted, it could be, but could also represent a personal choice to never interact with people after the cataclysm)

Deaf or mute characters can atleast wave to other NPCs to greet them/show they aren't hostile. Hold up a sign to let them know they can't speak (and this could work for cases of temporary deafness in game as well)

Crafting a writing tablet would be a straight forward tool - cardboard and a pen/pencil. It'd only work with those who are literate, and who are patient enough to communicate this way
More advanced tools like finding a TTS app on a USB stick, or some kind of speak and spell could be fancier solutions like transition lenses is for poor eyesight.

**Comment by KHeket** [comment]
> This isn't necessarily true. There are certain traits that enforce specific behaviors, such as hates vegetables forcing a dietary restriction for no reason other than "veggie icky" and not an allergy. Illiterate characters could have fun reading a comic book and could probably understand the Exodii blueprints, but the trait makes for better gameplay if it hard-disables books rather than having many special cases. Wayfarer prevents vehicle use. Brawler prevents guns and bows. Etc. It's probably OK for Mute to in game be a "your access to NPCs is totally disabled" trait.

So its the reason to make different pr, to remove or make this traits realism or lore friendly: psychology/psychiatry problem or trauma, portal-storm/mutagen effect or something else, but it must reasonable by something, not just "We will remove a lot of content or/and abilites just for fun or challenge". All must be close to realism, and lot of thing also was removed/reworker in that way. For more roguelike and fun we have Cataclysm: Bright Nights, or mods like magiclysm

**Comment by Zireael07** [comment]
> It would be totally acceptable if this was a trait about being anti-social and refusing to engage with other people - but this is a trait that is based on a real disability, a disability which people have learned to live with and interact meaningfully with their community daily.

> A new trait that is simply "anti-social" for characters that refuse to interact with NPCs is easily justifiable in the way a brawler refuses to use guns/bows. This could describe either a behavioral trait that isn't strictly associated to a particular disability (which granted, it could be, but could also represent a personal choice to never interact with people after the cataclysm)

As someone who has both mute and D/deaf friends IRL, quoting those two paragraphs for truth.

TLDR: make a new "anti-social" (or some other name that isn't a real-life disability that can be overcame easily with modern tech or some smarts - pen and paper anyone?) trait to represent "cannot communicate" and allow deaf/mute characters to communicate

**Comment by actually-the-artificer** [comment]
I agree; we should make an "anti-social" trait for "cannot communicate" and allow deaf/mute characters to communicate provided they have the proper tools.

**Comment by actually-the-artificer** [comment]
I agree. We should have anti-social as a "cannot communicate" trait and allow deaf/mute characters to communicate if they have the proper tools.

(whoops, accidentally closed this. Should be all good now.)

**Comment by Maleclypse** [comment]
Mute is an NPC only trait. How are you getting it? I’m inclined to close this since mute shouldn’t be accessible to the pc.

**Comment by actually-the-artificer** [comment]
It was part of a custom profession from my mod.

**Comment by actually-the-artificer** [comment]
It is accessible to the PC. See #43116.

**Comment by Maleclypse** [comment]
Yes that is a temporary assignment of the mute trait while your character is a werewolf and the entire design is that you temporarily cannot speak to NPCs at all. I’m not opposed to a different character accessible permanent trait that is not fully incapable of communication but it can’t be that trait.
